doc/oper.xml

```diff
@@ -1387,3 +1387,27 @@ gap> DigraphShortestDistance(gr, [1, 3], [3, 5]);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="DigraphClosure">
+<ManSection>
+  <Oper Name="DigraphClosure" Label="for a digraph and positive integer"
+        Arg="digraph, k"/>
+  <Returns>A digraph</Returns>
+  <Description>
+    Given a symmetric loopless digraph with no multiple edges <A>digraph</A>,
+    the <A>k</A>-closure of <A>digraph</A> is defined to be the unique smallest
```

Comment:
Since you're defining k-closure, I think you should enclose it in E tags: `<E><A>k</A>-closure</E>`